sdk/core/core-tracing/lib/plugins/opencensus/openCensusSpanPlugin.ts

```diff
@@ -13,23 +13,20 @@ export class OpenCensusSpanPlugin implements Span {
 
   constructor(span: any) {
     this._span = span;
+    this._span.start();
```

Comment:
Why is that we no longer care about the start time?

Reply:
I've been digging into this. It seems that while start() isn't there, there is some mention in the spec about it being possible to create spans "out of band" with custom timings: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-tracing.md

There is a tracking bug: https://github.com/open-telemetry/opentelemetry-specification/issues/139
And a proposal: https://github.com/open-telemetry/oteps/blob/master/text/0002-remove-spandata.md

But there *is* a way to do this today that I am going to adopt in this PR, which is that the Span constructor accepts a timestamp in the OpenTelemetry implementation: https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-basic-tracer/src/Span.ts#L56

Reply:
Also it turns out that OpenCensus doesn't support passing the timestamps:

https://github.com/census-instrumentation/opencensus-node/blob/ef5712fd3b279b0e80494322231232047b06f9e6/packages/opencensus-core/src/trace/model/span.ts

https://github.com/census-instrumentation/opencensus-node/blob/ef5712fd3b279b0e80494322231232047b06f9e6/packages/opencensus-core/src/trace/model/types.ts#L314

I think this was missed before when we used `any` instead of actual types.